packages/v4/src/content/design-guidelines/components/text-input/text-input.md

```diff
@@ -16,29 +16,29 @@ Use text input to gather free-form text from a user.
 
 ## Usage
 * Be sure to size input fields appropriately to anticipate the longest likely value (if a maximum character length is known).
-* When a value is too long to be fully displayed in a text field, use [truncation](/design-guidelines/content/grammar-and-terminology#truncation).
-* Provide a [text area](/documentation/react/components/textarea) for input anticipated to be longer than a single line.
+* When a value is too long to be fully displayed in a text field, use [truncation](/ux-writing/truncation).
+* Provide a [text area](/components/text-area) for input anticipated to be longer than a single line.
 * Pre-populate fields with known values when possible, like a default IP address, for example.
 
 ### Help
 **Placeholder text**
 
-[Placeholder text](/design-guidelines/usage-and-behavior/forms#placeholder-text) is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be "DD/MM/YYYY".
+[Placeholder text](/components/forms/design-guidelines#placeholder-text) is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be "DD/MM/YYYY".
 
 **Helper text**
 
-[Helper text](/design-guidelines/usage-and-behavior/forms#helper-text) is text below a form field that helps a user provide the right information, like "Enter a unique name".
+[Helper text](/components/forms/design-guidelines#helper-text) is text below a form field that helps a user provide the right information, like "Enter a unique name".
 
 
 **Popovers**
 
-[Popovers](/design-guidelines/usage-and-behavior/forms#popovers) provide context around a text or input field. Popovers are represented with an unfilled question mark con (fa-question-circle) and displayed when the user clicks on the icon.
+[Popovers](/components/forms/design-guidelines#popovers) provide context around a text or input field. Popovers are represented with an unfilled question mark con (fa-question-circle) and displayed when the user clicks on the icon.
 
-For more information about how to provide help content, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#user-help).
+For more information about how to provide help content, see  the [forms design guidelines](/components/forms/design-guidelines#user-help).
```

Comment:
Same here: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural